gitlab-pipeline/stage/test.yml

```diff
@@ -251,8 +251,13 @@ test:backend-integration:azblob:enterprise:
     - df -h . | tail -1 | awk '{system("hdparm -tT "$1);}'
 
     # Get and install the integration test requirements
-    - apk add $(cat ${WORKSPACE}/integration/tests/requirements/apk-requirements.txt)
-    - pip install -r ${WORKSPACE}/integration/tests/requirements/python-requirements.txt
+    - if [ -d ${WORKSPACE}/integration/tests/requirements-python ]; then
+    -   apk add $(cat ${WORKSPACE}/integration/tests/requirements-system/apk-requirements.txt)
```

Comment:
this can horribly break, but I assume you are aware of this. alternative:
```bash
while read -r; do apk add "$REPLY"; done < "${WORKSPACE}/integration/tests/requirements-system/apk-requirements.txt"
```

Reply:
Because of spaces in the names? Are there any such packages?

In any case, I think executing one at a time is not a good solution, it will be much slower. A better solution would be to set `IFS` to newline prior to command.

But honestly I wouldn't bother, since having a package with a space in it would probably break half the CI pipelines on Github, so I don't think anyone will attempt that.